src/components/select/select.js

```diff
@@ -370,29 +367,32 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
         if (angular.isString(disabled)) {
           disabled = true;
         }
+
         // Prevent click event being registered twice
         if (isDisabled !== undefined && isDisabled === disabled) {
           return;
         }
         isDisabled = disabled;
         if (disabled) {
-          element.attr({'tabindex': -1, 'aria-disabled': 'true'});
-          element.off('click', openSelect);
-          element.off('keydown', handleKeypress);
+          element
+            .attr({'aria-disabled': 'true'})
+            .removeAttr('tabindex')
```

Comment:
Angular doesn't keep track of the attributes, `attr` keeps the original value that was set.